### PR TITLE
LPS-175476 | Poshi Automation for Bug LPS-168698 | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-168698

Hi! :)
This pr is corresponding to this ticket: [LPS-168698](https://issues.liferay.com/browse/LPS-168698) → [LPS-175476](https://issues.liferay.com/browse/LPS-175476)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

**Review by Tim-Cao — COMMENTED** [review]
Hi @rafaellajordao, would you mind refactoring according to comments? Thanks

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesForWebContentArticle.testcase [hunk 1]
```diff
@@ -1501,6 +1501,49 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-168698. Expired Web Content Display Pages are not accessible"
+	@priority = 3
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagetemplate/displaypagetemplate/DisplayPageTemplatesForWebContentArticle.testcase:1505]: Should be 4 same with bug ticket

**Comment by Tim-Cao** [comment]
Good job @rafaellajordao

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**